step2-06/README.md

```diff
@@ -1,27 +1,29 @@
-# Step 2.6
+# Step 2.6 - Redux: Dispatching actions and examining state
```

Comment:
I thought this step was pretty good overall. It definitely helped me understand the basics of actions, reducers, and Redux-managed state. 

BUT if this was a good level for me, that makes me worried that it will be too much, too fast for someone who's just learning JS/React (especially someone from a non-dev background).